docs/usage/configuration-options.md

```diff
@@ -642,24 +642,6 @@ Descriptions fields embedded within presets are also collated as part of the onb
 
 Add to this object if you wish to define rules that apply only to PRs that update digests.
 
-## docker
```

Comment:
We're working on a new major version in the `v35` branch, so maybe this PR should target `v35`. Right now this PR targets `main`. I'll let the maintainers decide. 😉

Reply:
v35